HW4.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
+
+//Task #1
+
+detonatorTimer(3);
+// 3
+// 2
+// 1
+// BOOM!
+
+function detonatorTimer(delay) {
+
+    let timerId = setInterval(() => {if (delay > 0) {console.log(delay); delay -= 1;} else {console.log('BOOM!'); clearInterval(timerId); }}, 1000);
+	
+}
```

Comment:
```suggestion
function detonatorTimer(delay) {
    let timerId = setInterval(() => {
        if (delay > 0) {
            console.log(delay);
            delay -= 1;
        } else {
            console.log('BOOM!');
            clearInterval(timerId);
        }
    }, 1000);
}
```

Все добре, тільки форматуй, будь ласка, код)